enhancements/installer/aws-cross-account-dns-zone.md

```diff
@@ -0,0 +1,340 @@
+---
+title: aws-cross-account-dns-zone
+authors:
+  - "@padillon"
+reviewers:
+  - "@Miciah"
+  - "@jharrington22"
+  - "@mtulio"
+  - "@gcs278"
+approvers:
+  - "@mrunalp"
+  - "@zaneb"
+api-approvers:
+  - "@deads2k"
+  - "@JoelSpeed"
+creation-date: 2023-05-08
+last-updated: 2023-05-08
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/CORS-2613
+  - https://issues.redhat.com/browse/OCPBU-558
+see-also:
+replaces:
+superseded-by:
+---
+
+# AWS Shared VPC with Cross-account DNS Zones
+
+## Summary
+
+AWS [VPC sharing](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html) allows a VPC
+to be shared between multiple accounts. Accordingly, it is possible to configure a DNS private hosted
+zone in one account (Account A) while cluster resources are installed in a different
+account (Account B). This enhancement enables OpenShift installation with this cross-account shared-VPC
+architecture by providing a solution to handle authentication to the private-hosted-zone account. 
+
+## Motivation
+
+Organizations utilize AWS shared VPCs to enable some degree of resource sharing with participants while
+maintaining administrative privileges over others. Enabling users to install OpenShift with a private
+hosted zone from a separate account allows DNS resolution for the cluster on the shared network while
+maintaining the least-privilege security principle. An alternative solution would be for the OpenShift
+Installer to create a private hosted zone and attach the zone to the shared VPC. Instead, organizations
+can limit the required permissions and maintain greater administrative control by pre-creating a hosted
+zone and a role with limited permissions in that secondary account.
+
+### User Stories
+
+* As an OpenShift cluster admin, I want to install OpenShift on a shared VPC and utilize a private hosted
```

Comment:
This user story is missing the why, it says who and what they want to do, but doesn't explain the benefit, what are they trying to achieve